client/main.lua

```diff
@@ -280,44 +262,49 @@ RegisterNetEvent("qb-garages:client:VehicleList", function(data)
                     v.state = Lang:t("status.impound")
                 end
                 if type == "depot" then
-                    MenuGarageOptions[#MenuGarageOptions + 1] = {
-                        header = Lang:t('menu.header.depot', { value = vname, value2 = v.depotprice }),
-                        txt = Lang:t('menu.text.depot', { value = v.plate, value2 = currentFuel, value3 = enginePercent, value4 = bodyPercent }),
-                        params = {
-                            event = "qb-garages:client:TakeOutDepot",
-                            args = {
-                                vehicle = v,
-                                type = type,
-                                garage = garage,
-                                index = indexgarage,
-                            }
-                        }
+                    registeredMenu.options[#registeredMenu.options + 1] = {
+                        title = Lang:t('menu.header.depot', { value = vname, value2 = v.depotprice }),
+                        description = '',
+                        event = 'qb-garages:client:TakeOutDepot',
+                        args = {
+                            vehicle = v,
+                            type = type,
+                            garage = garage,
+                            index = indexgarage,
+                            },
+                        metadata = {
+                            {label = 'Plate', value = v.plate .. ' '},
+                            {label = 'Engine', value = enginePercent .. ' %'},
+                            {label = 'Fuel', value = currentFuel .. ' %'},
+                            {label = 'Body', value = bodyPercent .. ' %'},
+                        },
                     }
+                    
+
                 else
-                    MenuGarageOptions[#MenuGarageOptions + 1] = {
-                        header = Lang:t('menu.header.garage', { value = vname, value2 = v.plate }),
-                        txt = Lang:t('menu.text.garage', { value = v.state, value2 = currentFuel, value3 = enginePercent, value4 = bodyPercent }),
-                        params = {
-                            event = "qb-garages:client:takeOutGarage",
-                            args = {
-                                vehicle = v,
-                                type = type,
-                                garage = garage,
-                                index = indexgarage,
-                            }
-                        }
+                    registeredMenu.options[#registeredMenu.options + 1] = {
+                        title = Lang:t('menu.header.garage', { value = vname, value2 = v.plate }),
+                        description = '',
+                        event = 'qb-garages:client:TakeOutDepot',
```

Comment:
```suggestion
                        event = 'qb-garages:client:takeoutGarage',
```

Reply:
There is a typo when passing the event name to take the vehicle out.
`qb-garages:client:takeoutGarage`
It should be
`qb-garages:client:takeOutGarage`